lib/ExportsStorage.js

```diff
@@ -192,6 +203,17 @@ export default class ExportsStorage {
             reject(err);
             return;
           }
+
+          if (!additional) {
```

Comment:
I had to read the code a few times to understand what this boolean meant. A comment here could probably make things clearer. 🤔